### PR TITLE
add the None type to the ord argument of the vector_norm backend

### PR DESCRIPTION
**Pull request by kimbring2**
The issue is that the default of the ord argument of the **jax.numpy.linalg.norm** function is None.

Another framework usually has an int-type value for that. 

Therefore, it looks like a good idea to add a condition for dealing with the None value for the norm of all backend.

I also add the None type for the test function to ensure that.

### CODE REVIEW

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by ivy-seed** [comment]
This PR has been closed because it has been marked as stale for more than 7 days with no activity.
